app/views/shared/_maintenance_window_alert.html.erb

```diff
@@ -0,0 +1,22 @@
+<%
+  maintenance_window = MaintenanceWindow.new(
+    start: Figaro.env.acuant_maintenance_window_start,
+    finish: Figaro.env.acuant_maintenance_window_finish,
+    now: local_assigns[:now],
+  )
+%>
+<% if maintenance_window.active? %>
+  <%= render 'shared/alert', { type: 'warning', class: 'margin-bottom-2', text_tag: 'div' } do %>
+    <p>
+      <%= t('notices.maintenance.currently_under_maintenance_html',
+            finish: l(maintenance_window.finish,
+                      format: t('time.formats.event_timestamp_with_zone'))) %>
```

Comment:
Does this use the timezone of the user? Or the servers?

Reply:
this uses whatever timezone the object has attached to it, and in the "model" class I updated it to convert everything to Eastern by default https://github.com/18F/identity-idp/pull/4202/files#diff-cf15a0561b523b2e35dcfea21b77602d

Reply:
>in the "model" class I updated it to convert everything to Eastern by default https://github.com/18F/identity-idp/pull/4202/files#diff-cf15a0561b523b2e35dcfea21b77602d

Ah! I'd missed that.

👍